qualys-web-application-scanner/operations.py

```diff
@@ -1,15 +1,26 @@
-import base64
+""" Copyright start
```

Comment:
Change it to """
Copyright start
MIT License
Copyright (c) 2024 Fortinet Inc
Copyright end
"""